tools/mo/openvino/tools/mo/ops/scatter.py

```diff
@@ -41,11 +41,6 @@ def infer(node: Node):
         updates_shape = node.in_port(2).data.get_shape()
         assert input_shape is not None and updates_shape is not None and indices_shape is not None, \
             'The node "{}" input shape is None'.format(node_name)
-        assert len(input_shape) == len(indices_shape), 'data and indices inputs for node "{}" must be of the ' \
-            'same rank. Instead got {} and {}'.format(node_name, len(input_shape), len(indices_shape))
-        assert compatible_shapes(indices_shape, updates_shape), \
-            'updates and indices shapes for node "{}" must be equal. Instead got {} and {}.' \
-            ''.format(node_name, indices_shape, updates_shape)
```

Comment:
These checks should be done for `ScatterElementsUpdate`

Reply:
Thanks, done.